src/components/FileMetadata/index.spec.tsx

```diff
@@ -1,34 +1,44 @@
-import * as React from 'react';
 import { shallow } from 'enzyme';
+import * as React from 'react';
 
 import configureStore from '../../configureStore';
 import {
   actions as versionActions,
-  getVersionFile,
+  getVersionFile as _getVersionFile,
```

Comment:
We tend to the do the opposite, i.e. prefixing with `_` in the code and not using import aliases.